core/services/standardcapability/standard_capability.go

```diff
@@ -0,0 +1,95 @@
+package standardcapability
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/loop"
+	"github.com/smartcontractkit/chainlink-common/pkg/types/core"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services/job"
+	"github.com/smartcontractkit/chainlink/v2/plugins"
+)
+
+type StandardCapability struct {
+	log                logger.Logger
+	spec               *job.StandardCapabilitySpec
+	pluginRegistrar    plugins.RegistrarConfig
+	telemetryService   core.TelemetryService
+	store              core.KeyValueStore
+	capabilityRegistry core.CapabilitiesRegistry
+	errorLog           core.ErrorLog
+	pipelineRunner     core.PipelineRunnerService
+	relayerSet         core.RelayerSet
+
+	capabilityLoop *loop.StandardCapabilityService
+}
+
+func NewStandardCapability(log logger.Logger, spec *job.StandardCapabilitySpec,
+	pluginRegistrar plugins.RegistrarConfig,
+	telemetryService core.TelemetryService,
+	store core.KeyValueStore,
+	capabilityRegistry core.CapabilitiesRegistry,
+	errorLog core.ErrorLog,
+	pipelineRunner core.PipelineRunnerService,
+	relayerSet core.RelayerSet) *StandardCapability {
+
+	return &StandardCapability{
+		log:                log,
+		spec:               spec,
+		pluginRegistrar:    pluginRegistrar,
+		telemetryService:   telemetryService,
+		store:              store,
+		capabilityRegistry: capabilityRegistry,
+		errorLog:           errorLog,
+		pipelineRunner:     pipelineRunner,
+		relayerSet:         relayerSet,
+	}
+}
+
+func (s *StandardCapability) Start(ctx context.Context) error {
+
+	cmdName := s.spec.Command
+
+	cmdFn, opts, err := s.pluginRegistrar.RegisterLOOP(plugins.CmdConfig{
+		ID:  s.log.Name(),
+		Cmd: cmdName,
+		Env: nil,
+	})
+
+	if err != nil {
+		return fmt.Errorf("error registering loop: %v", err)
+	}
+
+	s.capabilityLoop = loop.NewStandardCapability(s.log, opts, cmdFn)
+
+	if err = s.capabilityLoop.Start(ctx); err != nil {
+		return fmt.Errorf("error starting standard capability service: %v", err)
+	}
+
+	if err = s.capabilityLoop.WaitCtx(ctx); err != nil {
+		return fmt.Errorf("error waiting for standard capability service to start: %v", err)
+	}
+
+	if err = s.capabilityLoop.Service.Initialise(ctx, s.spec.Config, s.telemetryService, s.store, s.capabilityRegistry, s.errorLog,
```

Comment:
What is the KV-store for?

Reply:
Same reason as per common, pasted here->

My understanding is that the standard capability is analogous to a generic plugin, and thus should have the same resources available to it, one of those being keyvalue store.  As I understand it the idea is that we give the capability/plugin writers as much functionality as possible to avoid them having to write custom implementations that would have to access core code directly.